crates/rand_core/RUSTSEC-2019-0035.toml

```
@@ -1,5 +1,6 @@
 [advisory]
 id = "RUSTSEC-2019-0035"
+alias = "GHSA-mmc9-pwm7-qj5w"
```

Whoops sorry, this needs to be:

```suggestion
aliases = ["GHSA-mmc9-pwm7-qj5w"]
```